### ISSUE TITLE
Are there any pre-trained reward classifier model parameters available for other versions of ResNet?

### ISSUE THREAD

**Issue by JinGyunJeong** [rail-berkeley/serl#83]
[wget https://github.com/rail-berkeley/serl/releases/download/resnet10/resnet10_params.pkl]

I noticed that only one parameter file for the pre-trained ResNet-10 model has been made publicly available. I would like to inquire if there are any pre-trained reward classifier model parameters available for other versions of ResNet.

Since all the provided scripts are implemented using JAX and Flax, I am unable to utilize other pre-trained models for testing purposes. Would it be possible for you to provide support or guidance regarding this issue?

Thank you in advance for your time and assistance. I look forward to your response.

**Comment by jianlanluo** [comment]
we currently don't have plan to support torch, you could import torch model and just get the embedding from it. you are welcome to open a PR
